### ISSUE TITLE
Color for the real time spelling check

### ISSUE THREAD

**Issue by byzheng** [rstudio/rstudio#4507]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.66
    OS Version      :  Windows
    R Version       :  3.5.0
### Steps to reproduce the problem

Enable the read time spelling checking

### Describe the problem in detail
The color is too light and cannot see in the screen 
![image](https://user-images.githubusercontent.com/1657381/54960713-8f138800-4fa9-11e9-9a73-53826cb19d52.png)

### Describe the behavior you expected

Darker color will be better.

**Comment by gtritchie** [comment]
From an accessibility standpoint, we'd definitely want a much higher contrast ratio for the squiggly line (and/or some additional UI affordance that is more visible for low-vision users).

**Comment by adamconroy** [comment]
I agree this is too low contrast. We are currently leaning on the "warning" color from the current user theme. There are a few options we can consider.

**Comment by adamconroy** [comment]
This is...better now. It's not incredible but kind of the best all-in-one solution we can do without really reworking how the squiggles are done (and being generic for light/dark themes). We are moderately limited by Ace's implementation at the moment.
